Emby.Server.Implementations/Data/BaseSqliteRepository.cs

```diff
@@ -97,9 +97,16 @@ protected BaseSqliteRepository(ILogger<BaseSqliteRepository> logger)
         /// </summary>
         /// <value>The write connection.</value>
         protected SQLiteDatabaseConnection WriteConnection { get; set; }
+        protected SQLiteDatabaseConnection ReadConnection { get; set; }
```

Comment:
```suggestion

        protected SQLiteDatabaseConnection ReadConnection { get; set; }
```

Newline between properties